lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -21,10 +21,21 @@ class BaseI18nTestCase(TestCase):
     """
     Base utilities for i18n test classes to derive from
     """
+    def __init__(self, *args, **kwargs):
```

Comment:
I will attempt to remove this again.  I did not intend to add this in the beginning, but the quality tests were failing without it.  Let me see if I can remove it again.